### ISSUE TITLE
NCBIXML does not set "bits" in 1.74

### ISSUE THREAD

**Issue by SchwarzMarek** [biopython/biopython#2179]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```python
import sys; print(sys.version)
import platform; print(platform.python_implementation()); print(platform.platform())
import Bio; print(Bio.__version__)

3.7.3 | packaged by conda-forge | (default, Jul  1 2019, 21:52:21) 
[GCC 7.3.0]
CPython
Linux-4.15.0-46-generic-x86_64-with-debian-buster-sid
1.74
```

### Expected behaviour

The `bits` attribute in the HSP object having a value (float).

### Actual behaviour

The `bits`attribute is set to `None`.

### Steps to reproduce

Obtain XML file from `biopython/Tests/Blast/xml_2212L_blastx_001.xml`.

Run attached code:
```
import sys
import Bio
from Bio.Blast import NCBIXML

print('Biopython version {}'.format(Bio.__version__))

with open('xml_2212L_blastx_001.xml', 'r') as f:
    for out in NCBIXML.parse(f):
        for hit in out.alignments:
            for hsp in hit.hsps:
                print(hsp.bits)
                sys.exit(0)
```

For biopython 1.74 output is `None`, while for 1.72 and 1.73, there is `247.284`.

**Comment by peterjc** [comment]
From the timing, this is most likely a side effect of adding BLAST XML v2 support in #1997, could you take a quick look @rtf-const?

**Comment by peterjc** [comment]
Fixed by #2180

**Comment by alffdtu** [comment]
This bug does not seem fixed in the Biopython that is installed with pip. I get the same error with the biopython 1.74 upgraded with pip

**Comment by peterjc** [comment]
@alffdtu Correct, this bug was reported against Biopython 1.74.

The fix ce2a57d1b9ec53ec5814d1de83bf89438bdfe747 was merged on 23 July, so will be included in Biopython 1.75.
